InnerEye/ML/SSL/lightning_containers/ssl_container.py

```diff
@@ -148,10 +148,12 @@ def create_model(self) -> LightningModule:
         # of a 7x7 conv layer.
         use_7x7_first_conv_in_resnet = False if self.ssl_training_dataset_name.value.startswith("CIFAR") else True
 
-        # Adjust the learning rate to work with multi gpu
+        # Rescale the learning rate linearly according to the number of available GPUs, as seen in: https://arxiv.org/abs/1706.02677,
+        # to avoid a drop in performance
         gpus_per_node = self.num_gpus_per_node()
-        l_rate = self.l_rate * self.num_nodes * (gpus_per_node if gpus_per_node > 0 else 1)
-        print(f"We found {self.num_nodes * gpus_per_node} GPUs, SSL encoder learning rate has been adjusted from {self.l_rate} to {l_rate}")
+        if self.num_nodes * gpus_per_node > 1:
```

Comment:
I'd precompute `self.num_nodes * gpus_per_node` into a variable, maybe called `batch_size_multiplier` or `num_gpus` or something similar, and replace the three multiplications.